request-cert/main.go

```diff
@@ -37,24 +37,30 @@ import (
 var (
 	certificateType = flag.String("type", "", "certificate type: node or client")
 
-	addresses = flag.String("addresses", "", "comma-separated list of DNS names and IP addresses for node certificate")
-	user      = flag.String("user", "", "username for client certificate")
+	addresses       = flag.String("addresses", "", "comma-separated list of DNS names and IP addresses for node certificate")
+	user            = flag.String("user", "", "username for client certificate")
 
+	cluster			= flag.String("cluster", "", "logical cluster name distinguishing it from other clusters in the same namespace")
 	namespace       = flag.String("namespace", "", "kubernetes namespace for this pod")
 	certsDir        = flag.String("certs-dir", "cockroach-certs", "certs directory")
-	keySize         = flag.Int("key-size", 2048, "RSA key size in bits")
+	keySize         = flag.Int("key-size", 4096, "RSA key size in bits")
 	symlinkCASource = flag.String("symlink-ca-from", "", "if non-empty, create <certs-dir>/ca.crt linking to this file")
 )
 
 func main() {
 	flag.Parse()
-	flag.Lookup("logtostderr").Value.Set("true")
```

Comment:
This line is removed  because it causes the following panic (on linux & macos) :

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1b46512]

goroutine 1 [running]:
main.main()
        /Users/conchr/Go/src/github.com/smartmachine/request-cert/request-cert/main.go:52 +0x92
```